docs/pages/reference/helm-reference.mdx

```diff
@@ -16,3 +16,6 @@ layout: tocless-doc
 - [teleport-plugin-jira](./helm-reference/teleport-plugin-jira.mdx): Deploy
   the Teleport Jira Access Request Plugin, which allows approving of denying
   Access Requests via a Jira Project.
+- [teleport-plugin-mattermost](./helm-reference/teleport-plugin-mattermost.mdx):
```

Comment:
This sounds like the Fluentd description—we should update it for Mattermost

Reply:
Thanks, I haven't noticed that!